### DISCUSSION TITLE
game launch and crash

### DISCUSSION THREAD

**Discussion by Plhorror** [termux/termux-app#3739] [Q&A] [answered]
hi,
i tested few games, they launch and crash immediately.
i tested all driver.
missing dll?

**Comment by twaik** [comment, marked as answer]
Probably that is wrong repo for this question. Also termux does not allow you to run anything related to dll's. Probably you are running something wine-related and it is not something we can help with, that is 3rd party software that we offer zero support for. It is not available in termux repos. 
In other words: Use at your own risk and we take zero responsibility for what that app does to Termux or your device.
